src/stratus-application-tests/src/test/java/org/geoserver/rest/catalog/RedisStyleTest.java

```diff
@@ -261,4 +267,14 @@ public void testPostToWorkspaceSLDPackage() throws Exception {
         Assert.assertNotNull(this.getCatalog().getResourceLoader().find("workspaces/gs/styles/foo.sld"));
     }
 
+    /**
+     * There is currently an issue with using file extensions / suffixes in the URL path, so this test is failing.
+     * Just bypass it until a solution is found.  https://github.com/spring-projects/spring-framework/issues/24041
```

Comment:
Maybe add a "TODO" here so this is a little easier to find/fix in the future?

Reply:
check